examples/README.md

```diff
@@ -0,0 +1,7 @@
+# cuDF Examples
+This directory contains examples of cuDF usage. Some examples are simple while others are more involved. Each example is held within a subdirectory. Each example will have its own README to explain intended use, assist with building, and running.
```

Comment:
```suggestion
This directory contains examples of cuDF usage.
```

Reply:
ahh I see you thought the same as I mentioned ^^ =)